README.md

```diff
@@ -48,3 +40,7 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 * [Why do contract testing?](docs/why-contract-testing.md)
 * [Running your frontend against the examples and random content (content-store not needed)](docs/running-frontend-against-examples.md)
 * [Deployment](docs/deployment.md)
```

Comment:
Optional: I think we can delete this file, since it only contains one line! We could explain this repo is deployed like any other app, but we still don't need a whole separate file to explain this.